Vehicle.h

```diff
@@ -12,18 +12,14 @@ struct Vehicle
     Vehicle(const Vehicle&) = default;
     Vehicle& operator=(const Vehicle&) = default;
     
-    virtual void setSpeed(int s)
-    {
-        std::cout << "\nHighway is adjusting " << name << "'s speed to " << s << " mph" << std::endl;
-        speed = s;
-    }
+    virtual void setSpeed(int s);
+    virtual void tryToEvade();
+    virtual void dontEvade();
```

Comment:
Don't add virtual functions that don't need to exist. 
remove `dontEvade`. the only one who uses it is the `SemiTruck`
- [ ] revise